src/olympia/stats/tests/test_views.py

```diff
@@ -7,41 +7,67 @@
 
 from django.http import Http404
 from django.test.client import RequestFactory
-from django.urls.exceptions import NoReverseMatch
 from django.utils.encoding import force_text
 
-from waffle.testutils import override_flag
-
 from olympia import amo
 from olympia.access.models import Group, GroupUser
 from olympia.addons.models import Addon, AddonUser
-from olympia.amo.tests import (TestCase, version_factory, addon_factory,
-                               user_factory)
-from olympia.amo.urlresolvers import reverse, resolve
+from olympia.amo.tests import (
```

Comment:
I had to rewrite most of the file because test cases verifying things like csv/json exports, etc. were based on the legacy stats. I kept almost everything and configured a BigQuery mock in each test method.